src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,13 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard
+        key={movie.imdbID}
```

Comment:
The property 'imdbID' should be 'imdbId' according to the task description. Make sure the case matches exactly to avoid potential bugs.